Changelog.md

```diff
@@ -2,6 +2,11 @@
 
 ### Ignition Rendering 4.X
 
+### Ignition Rendering 4.x.x (202x-xx-xx)
+
+1. Add support for 8 bit thermal camera image format
+    * [Pull Request #235](https://github.com/ignitionrobotics/ign-rendering/pull/235)
```

Comment:
we used to do this quite a bit in our PRs. I did it here just for convenience since we're planning on making another release soon. The `xx` are just placeholders